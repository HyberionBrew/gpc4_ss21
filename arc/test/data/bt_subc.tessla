in x : Events[Int]
def y = x - 3
out y
