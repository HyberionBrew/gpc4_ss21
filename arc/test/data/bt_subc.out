2: y = -2
5: y = 0
6: y = 6
9: y = 1
12: y = 21
13: y = 29
14: y = 40
17: y = -1
19: y = 20
20: y = 9
22: y = 21
23: y = 29
24: y = 40
27: y = -1
29: y = 20
30: y = 9
